xbmc/windowing/wayland/WinSystemWayland.cpp

```diff
@@ -774,7 +774,7 @@ void CWinSystemWayland::ProcessMessages()
   }
 }
 
-void CWinSystemWayland::ApplyShellSurfaceState(IShellSurface::StateBitset state)
+void CWinSystemWayland::ApplyShellSurfaceState(const IShellSurface::StateBitset& state)
```

Comment:
from what I am seeing the StateBitset is a std:::bitset<5> which is just 8 bytes, so it should be actually cheaper to pass by value then by reference?

Reply:
Yeah these changes do not really seem useful